src/pages/iou/IOUModal.js

```diff
@@ -315,12 +351,15 @@ class IOUModal extends Component {
                                     )}
                                     {currentStep === Steps.IOUConfirm && (
                                         <IOUConfirmPage
-                                            onConfirm={this.createTransaction}
+                                            onConfirm={this.state.iouType === 'send'
```

Comment:
@nickmurray47 Should we create some global enum for the `iouType` so we do not repeat the string everytime we check for this?

Reply:
Yes, that would be ideal